### ISSUE TITLE
Unicode decode error can occur from path(s) with "\x" in them, subprocess still uses backslashes

### ISSUE THREAD

**Issue by CabbageMcGravel** [bonjorno7/SourceOps#71]
Reposting comment from closed PR #70

It appears that if a path has `\x` in it, it is interpreted as a Unicode escape character instead of a path:
![image](https://user-images.githubusercontent.com/87149601/126368976-02420b06-bbdb-4ccd-a3c4-924b167d62fa.png)
Debugging seems to indicate that `Popen` makes the path with backslashes, even though the `args` variable appears to use forward ones as input?

![image](https://user-images.githubusercontent.com/87149601/126372357-ad8f12d4-9992-4a28-8614-ef9bf4c9b944.png)

https://github.com/bonjorno7/SourceOps/blob/bbc8838ca4a22df424b69d4816c761799d122b4b/addon/types/model_export/model.py#L313

More info on what is happening on Windows: https://docs.python.org/3/library/subprocess.html#converting-argument-sequence

_Originally posted by @CabbageMcGravel in https://github.com/bonjorno7/SourceOps/issues/70#issuecomment-883579738_

**Comment by CabbageMcGravel** [comment]
This is actually visible in the `.log` file too it seems.
```
qdir:    "d:\steamlibrary\steamapps\common\half-life 2\hl2\modelsrc\example\model"

gamedir: "D:\SteamLibrary\steamapps\common\Half-Life 2\hl2"

g_path:  "D:\SteamLibrary\steamapps\common\Half-Life 2\hl2\modelsrc\example\model\model.qc"

Building binary model files...

Working on "model.qc"

SMD MODEL Collection.smd

SMD MODEL model_anims.smd

---------------------

writing D:\SteamLibrary\steamapps\common\Half-Life 2\hl2\models/example/model.mdl:
...and so on...
```

**Comment by CabbageMcGravel** [comment]
#68 seems to change the affected line where the error occurs, I wonder if it would fix this issue?

**Comment by CabbageMcGravel** [comment]
I checked the above, the issue still occurs there unfortunately.
Besides that, it appears to be a functional PR!

**Comment by CabbageMcGravel** [comment]
@bonjorno7 Is this issue closed by commit 4ab6a8e?

**Comment by bonjorno7** [comment]
I think so yeah.
